scripts/rtinst

```diff
@@ -243,7 +243,13 @@ if [ "$relno" = "20" ] || [ "$(lsb_release -sr)" = "20.04" ] || [ "$relno" = "11
   libcver=libcurl4
 fi
 
-if [ "$relno" = "21" ] || [ "$(lsb_release -sr)" = "21.04" ]; then
+if [ "$relno" = "21" ] || [ "$(lsb_release -sr)" = "21.10" ]; then
+  phpver=php7.4
+  phploc=/etc/php/7.4
+  libcver=libcurl4
+fi
+
+if [ "$relno" = "22" ] || [ "$(lsb_release -sr)" = "22.04" ]; then
```

Comment:
Ubuntu 22.04 doesn't have php7.4 on its repository. So we can mention php8.1 without using any third-party repository for php7.4.

Reply:
php8.1 doesnt have geoip or an equivalent package but i'm sure you checked that

Reply:
> php8.1 doesnt have geoip or an equivalent package. This pull request will be removed shortly and a new one put up as i've found a way around it. Also it looks like this project isn't being updated much anyway

Yeah I know. But rtorrent will work flawlessly without this particular package! I have already tested that..

Reply:
Yes it will, but i want that feature